sofar/sofastream.py

```diff
@@ -101,6 +101,108 @@ def __getattr__(self, name):
 
         return self._data
 
+    @property
+    def list_dimensions(self):
+        """
+        Print the dimensions of the SOFA-file
+
+        See :py:func:`~SofaStream.inspect` to see the shapes of the data inside
+        the SOFA-file and :py:func:`~SofaStream.get_dimension` to get the
+        size/value of a specific dimensions as integer number.
+
+        The SOFA standard defines the following dimensions that are used
+        to define the shape of the data entries:
+
+        M
+            number of measurements
+        N
+            number of samles, frequencies, SOS coefficients
+            (depending on self.GLOBAL_DataType)
```

Comment:
I would omit the `self` everywhere. It's kind of techy and might confuse unexperienced users more than its helping.
```suggestion
            (depending on GLOBAL_DataType)
```